helm/kafka/scripts/create-keystore.sh

```diff
@@ -0,0 +1,44 @@
+set -e
+
+
+dir=/timecard-restapi-keystore
+alias=$1
+days=100
+password=$2
+bootstrap=$3
+
+cd $dir
```

Comment:
`mkdir -p ${keystore_dir} && cd "$_"` ?

Reply:
This assumes, that for, future services re-using this script, they should be aware that the mount path should follow the pattern <service_name>-keystore. Either that, or have the mount path as an additional script argument.
Either way, I think we should document this

Reply:
I have updated, to bring this in via the value files. I agree we should document this. My question would be where? would as part of the readme work for now until we centralise our helm charts.

Reply:
I suggest we add a temporary section for this under timecard README. This can be moved across to the centralised helm chart README when it is created

Reply:
Done